packages/importer/src/config.ts

```diff
@@ -147,8 +147,18 @@ export interface OneSchemaTemplateColumn {
   mapping_hints?: string[]
 }
 
+export interface OneSchemaTemplateColumnToAdd extends Omit<OneSchemaTemplateColumnToUpdate, "label"> {
```

Comment:
ah is the `Omit` necessary here since this type also defines label?

Reply:
Michael and I discussed that `Omit` is more idiomatic for typescript since it can help make it more clear that this field is being replace.